services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -23,14 +25,16 @@ import arrowLeftIcon from "assets/icons/arrows/icon_arrow_left_blue.png";
 import { getReportsForState } from "utils/api/requestMethods/report";
 
 export const DashboardPage = () => {
-  const navigate = useNavigate();
   const { userIsAdmin } = useStore().user ?? {};
   const { reportType, state } = useParams();
   const [isLoading, setIsLoading] = useState(true);
   const [reports, setReports] = useState<Report[]>([]);
   const { intro, body } = dashboardVerbiage;
   const fullStateName = StateNames[state as keyof typeof StateNames];
 
+  // TO-DO: update accordingly - if a user is an admin or a read-only type, use the selected state, otherwise use their assigned state
+  const activeState = state;
+
   useEffect(() => {
     if (!isReportType(reportType) || !isStateAbbr(state)) {
       return;
```

Comment:
Since reloadReports function is the same code as this chunk here, we can probably call reloadReports here so that we can cut down on duplicate code. This is more of a clean up than a necessity so i'm okay with it being a follow up on a different PR.

Reply:
addressed it in this one! thanks for catching that